packages/docregistry/src/context.ts

```diff
@@ -59,13 +59,18 @@ export class Context<
     const localPath = this._manager.contents.localPath(this._path);
     const lang = this._factory.preferredLanguage(PathExt.basename(localPath));
     this._model = this._factory.createNew(lang);
```

Comment:
We should pass here a new flag telling the factory if collaboration is enabled -> the model should use that information to set the appropriate `model.collaborative` properties.

```suggestion
    this._model = this._factory.createNew(lang, PageConfig.getOption('collaborative') === 'true');
```